go.mod

```diff
@@ -147,82 +146,83 @@ require (
 	gopkg.in/inf.v0 v0.9.1 // indirect
 	gopkg.in/natefinch/lumberjack.v2 v2.2.1 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	k8s.io/apiextensions-apiserver v0.32.1 // indirect
-	k8s.io/cloud-provider v0.32.0 // indirect
-	k8s.io/controller-manager v0.32.0 // indirect
-	k8s.io/cri-api v0.32.0 // indirect
-	k8s.io/cri-client v0.32.0 // indirect
-	k8s.io/dynamic-resource-allocation v0.32.0 // indirect
-	k8s.io/kms v0.32.0 // indirect
-	k8s.io/kube-openapi v0.0.0-20241212222426-2c72e554b1e7 // indirect
-	k8s.io/kube-scheduler v0.32.0 // indirect
-	k8s.io/kubectl v0.32.0 // indirect
-	k8s.io/kubelet v0.32.0 // indirect
-	k8s.io/mount-utils v0.32.0 // indirect
-	k8s.io/pod-security-admission v0.32.0 // indirect
+	k8s.io/apiextensions-apiserver v0.33.0 // indirect
+	k8s.io/cloud-provider v0.33.0 // indirect
+	k8s.io/controller-manager v0.33.0 // indirect
+	k8s.io/cri-api v0.33.0 // indirect
+	k8s.io/cri-client v0.33.0 // indirect
+	k8s.io/dynamic-resource-allocation v0.33.0 // indirect
+	k8s.io/kms v0.33.0 // indirect
+	k8s.io/kube-openapi v0.0.0-20250318190949-c8a335a9a2ff // indirect
+	k8s.io/kube-scheduler v0.33.0 // indirect
+	k8s.io/kubectl v0.33.0 // indirect
+	k8s.io/kubelet v0.33.0 // indirect
+	k8s.io/mount-utils v0.33.0 // indirect
```

Comment:
There is a bug in v0.33.0 for mount-utils, should we wati for v0.33.1? https://github.com/kubernetes/kubernetes/pull/131568

Reply:
The external provisioner should not use mount-utils at all, I think it's safe.